pkg/cmd/pulumi/state/state_unprotect.go

```diff
@@ -40,10 +40,12 @@ func newStateUnprotectCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "unprotect [resource URN]",
 		Short: "Unprotect resources in a stack's state",
-		Long: `Unprotect resource in a stack's state
+		Long: `Unprotect resource in a stack's state, allowing deletion
 
 This command clears the 'protect' bit on one or more resources, allowing those resources to be deleted.
 
+To undo this operation, use ` + "`pulumi reprotect`" + `on the resource URN.
```

Comment:
Perhaps just
```suggestion
To protect a resource, use ` + "`pulumi protect`" + `.
```
? And the equivalent opposite on the new command